apertium_neon_plugin/__init__.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, *args, **kwargs):
                    "https://www.apertium.org/apy/translate"
 
     def translate(self, text, target=None, source=None, url=None):
-        if self.boost and not source:
+        if self.__dict__.get("boost", None) and not source:
```

Comment:
Rather than casting to dict, it's a little cleaner to check:

```python
if hasattr(self, 'boost') and self.boost
```

It looks like this was probably a copy/paste error in the first place, the `LanguageTranslator` class doesn't define `boost`, only the `LanguageDetector`.. So for this case, lines 14-15 should be replaced with:

```python
source = source or self.default_language
```